scripts/simplify_network.py

```diff
@@ -522,6 +523,64 @@ def consense(x):
     return clustering.network, clustering.busmap
 
 
+def merge_isolated_buses(n, drop_p_threshold=0):
```

Comment:
this function can be split into two:
1. a function that actually only drops buses below an energy threashold
2. a function that aggregates buses below another energy threshold